src/DssSpell.t.sol

```diff
@@ -557,76 +561,83 @@ contract DssSpellTest is DssSpellTestBase {
         assertTrue(spell.done());
 
         // check new allowance
-        uint256 newVesting = 2_216.4  ether; // CHRONICLE_LABS; note: ether is a keyword helper, only MKR is transferred here
-               newVesting += 1_619.93 ether; // JETSTREAM; note: ether is a keyword helper, only MKR is transferred here
+        uint256 newVesting = 216 ether; // JANSKY; note: ether is a keyword helper, only MKR is transferred here
+               newVesting += 216 ether; // VOTEWIZARD; note: ether is a keyword helper, only MKR is transferred here
         assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + newVesting, "testVestMKR/invalid-allowance");
 
+        // Let's move this general check to a separate test case
         assertEq(vest.cap(), 2_220 * WAD / 365 days, "testVestMKR/invalid-cap");
         assertEq(vest.ids(), prevStreamCount + 2, "testVestMKR/invalid-stream-count");
 
-        { // check CHRONICLE_LABS stream
-            address chronicleAddress = wallets.addr("CHRONICLE_LABS");
-            uint256 chronicleStreamId = prevStreamCount + 1;
-            uint256 chronicleFin = JUL_01_2023 + 366 days - 1;
-            assertEq(vest.usr(chronicleStreamId), chronicleAddress, "testVestMKR/invalid-address");
-            assertEq(vest.bgn(chronicleStreamId), JUL_01_2023, "testVestMKR/invalid-bgn");
-            assertEq(vest.clf(chronicleStreamId), JUL_01_2023, "testVestMKR/invalid-clif");
-            assertEq(vest.fin(chronicleStreamId), chronicleFin, "testVestMKR/invalid-calculated-fin");
-            assertEq(vest.fin(chronicleStreamId), JUN_30_2024, "testVestMKR/invalid-fin-variable");
-            assertEq(vest.mgr(chronicleStreamId), address(0), "testVestMKR/invalid-manager");
-            assertEq(vest.res(chronicleStreamId), 1, "testVestMKR/invalid-res");
-            assertEq(vest.tot(chronicleStreamId), 2_216.4 ether, "testVestMKR/invalid-total"); // note: ether is a keyword helper, only MKR is transferred here
-            assertEq(vest.rxd(chronicleStreamId), 0, "testVestMKR/invalid-rxd");
-            prevBalance = gov.balanceOf(chronicleAddress);
+        { // check JANSKY stream
+            address janskyAddress = wallets.addr("JANSKY");
+            uint256 janskyStreamId = prevStreamCount + 1;
+            uint256 janskyFin = OCT_01_2023 + 366 days - 1;
```

Comment:
Add a comment explaining we are only using 366 days because there's a leap year adjustment in between.